ECommerce-Website/client/src/Components/Home/Banner.jsx

```diff
@@ -3,7 +3,7 @@ import { styled } from '@mui/material';
 
 import { bannerData } from '../../constant/data';
 import Carousel from 'react-multi-carousel';
-
+import 'react-multi-carousel/lib/styles.css';
```

Comment:
Why do we need this, we have already imported this in other component and that will work globally.